lib/list_module.rb

```diff
@@ -0,0 +1,7 @@
+module List
+  def list_dishes(data)
+    data.each_with_index do |dish, i|
+      puts "#{i+1}: #{dish[:name]} costs #{dish[:price]}"
```

Comment:
`puts` should only be used for debugging.  Construct and return a string here instead.  If some other piece of code want to `puts` that string, it can.